content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -23,12 +29,6 @@ This page describes the RuntimeClass resource and runtime selection mechanism.
 RuntimeClass is a beta feature (as of v1.14) for selecting the container runtime configuration to
 use to run a pod's containers.
 
-{{< warning >}}
-RuntimeClass includes *breaking* changes in the beta upgrade in v1.14. If you were using
-RuntimeClass prior to v1.14, see [Upgrading RuntimeClass from Alpha to
-Beta](#upgrading-runtimeclass-from-alpha-to-beta).
-{{< /warning >}}
-
 ### Set Up
 
 Prerequisite: Ensure the RuntimeClass feature gate is enabled (it is by default). See [Feature
```

Comment:
nit: not sure if you need `Prerequisite:`

Reply:
nit: line 44 the link labeled `below` to (#cri-documentaiton) needs to be changed to (#cri-documentation). After the change, does the link work?
nit: See the corresponding documentation ... to learn how to configure your CRI (Container Runtime Interface) implementation.

Reply:
Done, but moved acronym expansion to the previous sentence.